crypto_helpers/ecdsa/ecdsa.go

```diff
@@ -14,21 +14,36 @@
  *    limitations under the License.
  */
 
-package pulsar
+package ecdsa
 
 import (
 	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
 	"crypto/x509"
+	"encoding/asn1"
 	"encoding/pem"
+	"math/big"
 
 	"github.com/pkg/errors"
+	"golang.org/x/crypto/sha3"
 )
 
+// MakeHash makes hash from seed
+func MakeHash(seed []byte) [32]byte {
```

Comment:
Are you sure, that sha3 hash should a part of ecdsa helper? As for me, It's more logical to move the method to another file